contributing/data-model-schema.txt

```diff
@@ -29,9 +29,32 @@ wastes your time. From the top-level :file:`bioformats` folder, while
 the most thorough is :command:`git clean -dfx`.
 
 
+Major or minor release?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A minor release of the OME model schema may suffice for changes like
```

Comment:
Thanks a lot for that. As we are using `may/must`, I would propose to add the usual reference to [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for the exact interpretation of the terms.